quickstart/src/index.js

```diff
@@ -7,32 +7,50 @@ var previewTracks;
 var identity;
 var roomName;
 
-// Attach the Tracks to the DOM.
-function attachTracks(tracks, container) {
-  tracks.forEach(function(track) {
-    container.appendChild(track.attach());
-  });
+
+// Attach given track to the DOM.
+function attachTrack(container, track) {
+  container.appendChild(track.attach());
 }
 
-// Attach the Participant's Tracks to the DOM.
-function attachParticipantTracks(participant, container) {
-  var tracks = getTracks(participant);
-  attachTracks(tracks, container);
+// Attach array of Tracks to the DOM.
+function attachTracks(container, tracks) {
```

Comment:
@makarandp0 Nitpick: Since the function is about the Tracks, I think it should come first.